local-cli/generator/templates/index.android.js

```diff
@@ -28,9 +27,9 @@ var <%= name %> = React.createClass({
       </View>
     );
   }
-});
+}
 
-var styles = StyleSheet.create({
+const styles = StyleSheet.create({
```

Comment:
if you declare a variable with `const`, it cannot be used before declaring, unlike `var`. so you will need to move it to top.